### PR TITLE
detect-hostbits/v1: convert unittests

### PR DESCRIPTION
**Pull request by TheKharleeci**
Adds test for:
- Detect-hostbits-01: HostBitsTestSig05
- Detect-hostbits-01: HostBitsTestSig07
- Detect-hostbits-01: HostBitsTestSig08

Link to redmine ticket: https://redmine.openinfosecfoundation.org/issues/4911

### CODE REVIEW

File: tests/detect-hostbits/detect-hostbits-01/test.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+args:
+- -k none
+
+checks:
```
Comment (victorjulien) [inline tests/detect-hostbits/detect-hostbits-01/test.yaml:4]: we should probably also add checks for sid 1 and sid 2 not generating alerts, so `count: 0`

File: tests/detect-hostbits/detect-hostbits-02/test.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+args:
+- -k none
+
+checks:
```
Comment (victorjulien) [inline tests/detect-hostbits/detect-hostbits-02/test.yaml:4]: can you add a filter for sid 1 here as well?

File: tests/detect-hostbits/detect-hostbits-03/test.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+args:
+- -k none
+
+checks:
+- filter:
```
Comment (victorjulien) [inline tests/detect-hostbits/detect-hostbits-03/test.yaml:5]: sid 4 should get a check

**Review by victorjulien — CHANGES_REQUESTED** [review]
see comments

**Comment by TheKharleeci** [comment]
Followed by: #666

**Pull request closed without merging**